web/controllers/pair_retro_controller.ex

```diff
@@ -92,4 +114,10 @@ defmodule Pairmotron.PairRetroController do
       pair -> Pairmotron.Calendar.first_date_of_week(pair.year, pair.week)
     end
   end
+
+  defp redirect_and_flash_error(conn, message) do
+    conn
+    |> put_flash(:error, message)
+    |> redirect(to: pair_path(conn, :index))
```

Comment:
Unfortunately, we don't reliable have access to the year and week here. I could do a case statement in the redirect_and_flash_error if you like that redirects differently if the week and year are available?